metric.go

```diff
@@ -50,9 +50,20 @@ func NewMetricFamily(logContext string, mc *config.MetricConfig, constLabels []*
 		labels = append(labels, mc.ValueLabel)
 	}
 
+	// Create a copy of original slice to apply sort properly
+	sortableLabels := append(constLabels[:0:0], constLabels...)
```

Comment:
This will append the slice to itself, it doesn't change anything. It should instead be:
```suggestion
	sortedLabels := append([]int(nil), constLabels...)
```
See https://play.golang.org/p/c-_2YcYxi5p

Reply:
Yeah, it makes sense. I just came across the suggestion [here](https://github.com/go101/go101/wiki/How-to-perfectly-clone-a-slice%3F) and sort of liked the explanation behind the way I used in terms of consistency, so decided to try something else and also get your opinion about it.

It gives me the same result if I'm not missing anything: https://play.golang.org/p/rs0LoTAI8xY

Reply:
You're right, of course, please leave it as is. My apologies.

I did try your code in the playground, but I must have appended the value directly to the `[:0:0]` slice instead of appending the contents of the slice itself first.

Reply:
No worries, thanks for double checking!